pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/OfflineClusterIntegrationTest.java

```diff
@@ -3476,7 +3476,7 @@ public void testExplainPlanQueryV2()
         + "            PinotLogicalTableScan(table=[[default, mytable]])\n");
     assertEquals(response1Json.get("rows").get(0).get(2).asText(), "Rule Execution Times\n"
         + "Rule: AggregateProjectMergeRule -> Time:*\n"
-        + "Rule: Project -> Time:*\n"
+        + "Rule: EvaluateProjectLiteralRule -> Time:*\n"
```

Comment:
(minor) Consider changing the `HashMap` into a `LinkedHashMap` to preserve the order of rule application, and also make this deterministic